spacy/pipeline/edit_tree_lemmatizer.py

```diff
@@ -91,6 +97,7 @@ def __init__(
         overwrite: bool = False,
         top_k: int = 1,
         scorer: Optional[Callable] = lemmatizer_score,
+        store_activations=False,
```

Comment:
```suggestion
        store_activations: Union[bool, List[str]] = False,
```
Also needs to be documented in the doc string

Reply:
(there's some other places throughout the PR where I think the type could be added similarly)

Reply:
The issue is that MyPy then infers that `store_activations` is a `Union[bool, List[str]]`. But maybe I should get rid of this setter and replace it by a `set_store_activations` method.

Reply:
Replaced all setter uses by a `set_store_activations` method and added the type to the `__init__` signatures.